lib/services.template.ejs

```diff
@@ -252,6 +252,18 @@ module.factory(
 <%    }); // forEach methods name -%>
 <% } // for each scope -%>
 
+<% if (meta.modelSchema) { -%>
+    /**
+     * @ngdoc object
+     * @name <%-: moduleName %>.<%- modelName %>#schema
+     * @propertyOf <%-: moduleName %>.<%- modelName %>
+     * @description
+     * The schema of the model represented by this $resource
+     */
+    R.schema = <%- JSON.stringify(meta.modelSchema, null, '\t') -%>
```

Comment:
Please use 2 spaces for indentation for consistency, we don't use tabs anywhere else.